policyengine_us/parameters/gov/states/ar/tax/income/exemptions/exemptions.yaml

```
@@ -1,16 +1,6 @@
 description: Arkansas accounts for these exemptions when calculating the state adjusted gross income.
```

Is this accurate? Is it AGI or gross income?
```suggestion
description: Arkansas exempts these sources from state adjusted gross income.
```

---

should be gross income